apps/ui/src/views/Proposal.vue

```diff
@@ -317,6 +317,13 @@ watchEffect(() => {
             :decimals="votingPowerDecimals"
           />
         </div>
+        <div>
+          <h4 class="mb-2.5 eyebrow flex items-center">
+            <IH-clock class="inline-block mr-2" />
+            <span>Timeline</span>
+          </h4>
+          <ProposalTimeline :data="proposal" />
+        </div>
```

Comment:
```suggestion
        <div>
          <h4 class="mb-2.5 eyebrow flex gap-2 items-center">
            <IH-clock />
            <span>Timeline</span>
          </h4>
          <ProposalTimeline :data="proposal" />
        </div>
```

Reply:
I was reusing the same as "results" 🥲  
<img width="596" alt="Untitled 4" src="https://github.com/user-attachments/assets/3c7f97c5-bccd-468e-b04e-a923a03c3a46">

Also 

<img width="493" alt="Untitled 5" src="https://github.com/user-attachments/assets/713c3d7f-1e29-466e-9700-0c292109843c">


should we keep it the same everywhere?

Reply:
I'm fine with either, we could keep it as-is and if we decide that we want to unify we could do it in one go later.

Reply:
Hmm yes, better I will change all in a separate PR